hist/hist/src/THStack.cxx

```diff
@@ -270,7 +270,7 @@ THStack::THStack(TH1* hist, Option_t *axis /*="x"*/,
          else if (sAxis.First('z')==kNPOS)
             haxis=hist->GetZaxis();
          if (!haxis) {
-            Warning("HStack","Histogram axis is NULL");
+            Warning("THStack","Histogram axis is NULL");
```

Comment:
Can a 1D histogram have a Z axis ? I am not sure why there is a check here on the z axis

Reply:
I thought of it and made a test with 1D. it does not crash or complain. But I can issue a Warning in case of 1 D